docs/in-depth/decky-loader.md

```diff
@@ -13,3 +10,12 @@ To enable *Decky Loader* in your configuration:
   };
 }
 ```
+
+### Setup
+
+After having switched to your new configuration with Decky enabled, open Steam, navigate to
+**Settings > Developer** and activate the `CEF Remote Debugging` setting. This is required for
```

Comment:
IIRC, the Developer settings menu itself is locked behind a switch in another settings menu.

Reply:
Possible, but I don't remember enabling anything to make that show up. Do you have any idea where that setting might be?

Reply:
I did not see Settings > Developer. I googled it and found https://help.gamemaker.io/hc/en-us/articles/11061313430045-Deploying-Games-to-a-Steam-Deck. You need to go into System > Enable Developer Mode